crux-http-server/src/crux/http_server/entity.clj

```diff
@@ -1,23 +1,22 @@
 (ns crux.http-server.entity
-  (:require [crux.http-server.util :as util]
-            [crux.http-server.entity-ref :as entity-ref]
-            [cognitect.transit :as transit]
-            [clojure.edn :as edn]
+  (:require [clojure.edn :as edn]
             [clojure.instant :as instant]
+            [clojure.java.io :as io]
             [clojure.string :as string]
-            [clojure.set :as set]
-            [crux.api :as api]
+            [cognitect.transit :as transit]
+            [crux.api :as crux]
```

Comment:
This is somewhat inconsistent with the naming elsewhere (see `http-server.clj` and the various other files in the `http-server` folder) - likely worth deciding if, within this section of the code, we should refer to crux calls as `crux/...` or `api/...`

Reply:
I'm trying to standardise on `crux.api :as crux`, because this is what I'd like it to be in any example code - our users won't be requiring `crux.api :as api`.

Reply:
Sounds good - likely worth us going through and ensuring that's consistent everywhere else?